test/Groups/directproducts.jl

```diff
@@ -181,6 +205,9 @@ end
    W = wreath_product(C,C,a)
    @test W isa WreathProductGroup
    @test order(W)==81
+   @test isisomorphic(factor_of_wreath_product(W,1),C)[1]
```

Comment:
But it shouldn't just be isomorphic, they should be equal, no?

Reply:
Unfortunately no. This is probably due to the fact that in GAP, the instruction `CyclicGroup(3)=CyclicGroup(3)` returns `false` (and this is the behaviour of `FpGroup` and `PcGroup` in GAP.